rotkehlchen/tests/unit/test_trades.py

```diff
@@ -127,6 +127,8 @@ def test_trades_from_dictlist(function_scope_messages_aggregator):
     assert isinstance(trades[0], Trade)
 
 
+@pytest.mark.xfail()
```

Comment:
@LefterisJP I've marked it on purpose because. It makes sense it fails because I'm not adding the new `custom_identifier` field in the `Trade` serialization (being cautious with backwards compatibility). I'd like to know the implications of it failing.

Reply:
@LefterisJP I've checked the CI job and more tests fail. Definitely I need to understand the implications of having modified Trade

Reply:
Some tests may fail (and did) due to 502 errors from one of the services we queried

Reply:
I actually no longer see any miscellaneous failing tests. (Last run passed with all green). So resolving this.